src/Analyzers/CSharp/Analyzers/AddBraces/CSharpAddBracesDiagnosticAnalyzer.cs

```diff
@@ -22,6 +22,7 @@ internal sealed class CSharpAddBracesDiagnosticAnalyzer :
     {
         public CSharpAddBracesDiagnosticAnalyzer()
             : base(IDEDiagnosticIds.AddBracesDiagnosticId,
+                   EnforceOnBuildValues.AddBraces,
```

Comment:
Is there a good way to guarantee that new analyzers will put constants in `EnforceOnBuildValues` instead of directly using `EnforceOnBuild` enum?

Reply:
Not really - this is the same problem as analyzers not defining constants in `IDEDiagnosticIds` class and directly inlining the IDExxxx constant. Hopefully, most new analyzers will see common usage patterns in existing analyzers which define similarly named constants in `IDEDiagnosticIds` and `EnforceOnBuildValues` and do the same.